rnd/autogpt_builder/src/components/ui/avatar.tsx

```diff
@@ -0,0 +1,50 @@
+"use client"
```

Comment:
Everything in components/ui is from [shadcn/ui](https://ui.shadcn.com/) ?

Reply:
yep